tests/mman/host/host.c

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/error.h>
+#include <openenclave/internal/tests.h>
+#include "mman_u.h"
+
+int main(int argc, char** argv)
+{
+    oe_result_t result;
+    oe_enclave_t* enclave = NULL;
+    int return_val = 0;
+
+    if (argc != 2)
+    {
+        fprintf(stderr, "Usage: %s ENCLAVE_PATH\n", argv[0]);
+        return 1;
+    }
+
+    const uint32_t flags = oe_get_create_flags();
+
+    if ((result = oe_create_mman_enclave(
+             argv[1], OE_ENCLAVE_TYPE_SGX, flags, NULL, 0, &enclave)) != OE_OK)
```

Comment:
why TYPE_SGX instead of TYPE_AUTO?  I don't see any other SGX-only code in this file.
```suggestion
             argv[1], OE_ENCLAVE_TYPE_AUTO, flags, NULL, 0, &enclave)) != OE_OK)
```

Reply:
Good point. Done.